// vitest/dist/browser.js
export { c as createExpect, d as describe, e as expect, i as it, s as suite, t as test } from './chunk-runtime-chain.4e2aa823.js';
import { e as dist } from './chunk-utils-import.16d9fb0d.js';
export { a as afterAll, d as afterEach, f as assertType, b as beforeAll, c as beforeEach, o as onTestFailed } from './chunk-utils-import.16d9fb0d.js';
export { a as setupGlobalEnv, s as startTests } from './chunk-runtime-setup.56d71d30.js';
import * as chai from 'chai';
export { chai };
export { assert, should } from 'chai';
import 'util';
import './chunk-mock-date.149ed990.js';
import 'node:path';
import 'picocolors';
import 'local-pkg';
import './chunk-utils-env.f4a39d2c.js';
import 'node:url';
import 'path';
import './vendor-_commonjsHelpers.addc3445.js';
import './chunk-utils-timers.52534f96.js';
import './vendor-index.723a074f.js';
import 'tinyspy';
import './chunk-utils-source-map.4e9b891d.js';
import './chunk-runtime-rpc.25cc9413.js';
import 'node:fs';
import 'fs';
import 'perf_hooks';
import './chunk-integrations-coverage.44413252.js';
import './chunk-runtime-error.97854396.js';
import 'vite-node/source-map';
import './chunk-env-node.b3664da2.js';
import 'node:console';



var expectTypeOf = dist.expectTypeOf;
export { expectTypeOf };
